python/lsst/obs/base/instrument_tests.py

```diff
@@ -103,6 +103,9 @@ def test_register(self):
         registeredInstrument = Instrument.fromName(self.instrument.getName(), registry)
         self.assertEqual(type(registeredInstrument), type(self.instrument))
 
+        # Check that re-registration is not an error.
+        self.instrument.register(registry)
```

Comment:
Does this mean that the try block at https://github.com/lsst/obs_base/blob/master/python/lsst/obs/base/gen2to3/convertRepo.py#L497-L500 can be removed now?

Reply:
It does indeed.  Done.